test/e2e/mapml-viewer/mapml-viewerAttributes.test.js

```diff
@@ -0,0 +1,37 @@
+import { test, expect, chromium } from '@playwright/test';
+
+test.describe("Adding Width and Height Attribute to mapml-viewer", () => {
+  let page;
+  let context;
+  test.beforeAll(async function() {
+    context = await chromium.launchPersistentContext('');
+    page = context.pages().find((page) => page.url() === 'about:blank') || await context.newPage();
+    await page.goto("mapml-viewerAttributes.html");
+  });
+  test.afterAll(async function () {
+    await context.close();
+  });
+
+  test("Setting Opacity Attibute to mapml-viewer", async () => {
+    let height_attribute_value = await page.$eval("body > mapml-viewer",(val) => val.getAttribute("height"));
+    let width_attribute_value = await page.$eval("body > mapml-viewer",(val) => val.getAttribute("width"));
+    let height;
+    let width;
+    if (!height_attribute_value && !width_attribute_value){
```

Comment:
we don't return from tests without testing our expectations!  That makes us think the test passed but it didn't actually test anything.